app.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, jsonify, request, abort
+from flask_cors import CORS
+import logging
+from cw_libs import clearwaters_docker as cwd
+import commands
+
+app = Flask(__name__)
+CORS(app)
+PORT=5656
+docker_client = cwd.CWDockerClient()
+used_ports = 8889
```

Comment:
Might be better to use a Flask application global rather than a global variable

http://flask.pocoo.org/docs/0.12/api/#application-globals

Reply:
More generally, I'm a bit concerned that we're not persisting `used_ports` somewhere (or getting an allocated port map from Docker or some other source of truth). If the server restarts, presumably there may be collisions if one tries to create a container on a previously allocated port.